templates/includes/sidebar.html

```diff
@@ -0,0 +1,15 @@
+<ul>
+  <li>
+    <a href="#">Home page</a>
+  </li>
+
+  <li>
+    <a href="#">Manufacturers</a>
+  </li>
+  <li>
+    <a href="#">Cars</a>
```

Comment:
The link for 'Cars' currently uses a placeholder URL ('#'). Consider updating this to point to the actual cars page URL.